west.yml

```diff
@@ -88,7 +88,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 1d823ea49bf7f6229a1097fbff33e27ba0dd389b
+      revision: pull/104/head
```

Comment:
The hal PR has been merged, so you can now update this to the new SHA

Reply:
Updated `west.yml`